3-enrich/spark-enrich/project/Dependencies.scala

```diff
@@ -15,27 +15,23 @@ import sbt._
 object Dependencies {
 
   val resolutionRepos = Seq(
-    // Required for our json4s snapshot
-    "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/",
-    // For some misc Scalding and Twitter libs
-    "Concurrent Maven Repo" at "http://conjars.org/repo",
-    "Clojars Maven Repo" at "http://clojars.org/repo",
-    // For Twitter libs
-    "Twitter maven libs" at "http://maven.twttr.com/",
     // For Snowplow libs
     "Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/",
```

Comment:
AFAIK, sce, collector-payload-1 and snowplow-thrift-raw-event are not published on central.